lib/views/pages/dashboard.dart

```diff
@@ -29,58 +33,78 @@ class Dashboard extends HookConsumerWidget {
           ),
         ],
       ),
       isTitleCenter: false,
       content: Container(
         alignment: Alignment.center,
-        child: Column(
-          children: [
-            TransactionSummary(
-              totalBalance: totalIncome - totalExpenses,
-              totalIncome: totalIncome,
-              totalExpenses: totalExpenses,
-              currency: currency,
-            ),
-            FilledButtonText(
-              text: 'Go To Notifications Page',
-              onPressed: () => {navigateTo(context, Routes.notifications)},
-            ),
-            OutlinedButtonText(
-              text: 'Outlined Pink',
-              color: AppColor.pink,
-              onPressed: () => {},
-            ),
-            OutlinedButtonText(
-              text: 'Outlined Blue',
-              onPressed: () => {},
-            ),
-            ElevatedButton(
-              onPressed: () => {navigateTo(context, '/account_settings')},
-              child: const Text('Go To Settings'),
-            ),
-            Visibility(
-              visible: user.username != 'User',
-              child: ElevatedButton(
-                onPressed: () => logout(ref),
-                child: const Text('Logout'),
+        child: Container(
+          margin: EdgeInsets.symmetric(horizontal: 10.0),
+          child: Column(
+            children: [
+              TransactionSummary(
+                totalBalance: widget.totalIncome - widget.totalExpenses,
+                totalIncome: widget.totalIncome,
+                totalExpenses: widget.totalExpenses,
+                currency: widget.currency,
+              ),
+              ProgressBar(
+                progress: spendingAmount,
+                label: "You have spent",
               ),
-            )
-          ],
+              Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Flexible(
+                    child: ElevatedButton(
+                      onPressed: () {
+                        setSpendingAmount(ref, spendingAmount - 0.1);
+                      },
+                      child: const Text('-'),
+                    ),
+                  ),
+                  const SizedBox(width: 30),
+                  Flexible(
+                    child: ElevatedButton(
+                      onPressed: () {
+                        setSpendingAmount(ref, spendingAmount + 0.1);
+                      },
+                      child: const Text('+'),
+                    ),
+                  )
+                ],
+              ),
+              FilledButtonText(
+                text: 'Go To Notifications Page',
+                onPressed: () => {navigateTo(context, Routes.notifications)},
+              ),
+              ElevatedButton(
+                onPressed: () => {navigateTo(context, '/account_settings')},
+                child: const Text('Go To Settings'),
+              ),
```

Comment:
Remove the sample buttons for **Go To Notifications Page** and **Go To Settings**